backend/models/organizations/orgs-model.js

```diff
@@ -1,4 +1,26 @@
-// const mongoose = require("mongoose")
-// // const { Schema } = mongoose
+const mongoose = require("mongoose")
+const { Schema } = mongoose
 
-// Add your code here
+const orgSchema = new Schema({
+    name: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    url: String,
+    tags: [String],
```

Comment:
We will use list of tag instances instead of string. If we have a large chunk of data, there will be a lot of copies of string with the same tag name. However, if we use tag instances, no matter how many number of instances of other data models, they only use references of unique tags. One unique tag exists in database for every tags. (This is a Many-To-Many relationship)

Recommend to address issue #64 first.